packages/errors/package.json

```diff
@@ -13,7 +13,7 @@
     "build:cjs": "babel --extensions \".js,.ts,.tsx\" --root-mode upward src --out-dir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
     "build:esm": "cross-env ESM=true babel --extensions \".js,.ts,.tsx\" --root-mode upward src --out-dir dist/esm",
-    "build:svg": "svgr --filename-case kebab --ext \"dist.js\" -d src/react/icons src/svg",
+    "build:svg": "svgr --typescript --filename-case kebab --ext \"dist.tsx\" -d src/react/illustrations src/svg",
```

Comment:
Your call, but for a single illustration, I wonder if we don't just prebuild and commit.  This scale feels different than, say, Icon, where we might get updates for 100s of icons and thus want to maintain only the raw svgs.  Seems like the svg build pipelines have been historically brittle, so I'm wondering if we can avoid one, maybe we should?  Anyway, that's just my line of thinking right now.

Reply:
Yeah we can probably get rid of this